intellij/src/saros/intellij/eventhandler/filesystem/LocalFilesystemModificationHandler.java

```diff
@@ -697,7 +721,11 @@ private void generateFileMove(
 
       editorManager.removeAllEditorsForPath(oldFilePath);
 
-      annotationManager.removeAnnotations(oldFilePath.getFile());
+      IFile file =
```

Comment:
Why are you calculating this `IFile` object even though it is you never used?

Reply:
Because...I don't know why. I removed this useless calculation...